crates/matrix-sdk-base/src/store/memory_store.rs

```diff
@@ -433,10 +433,7 @@ impl MemoryStore {
 
     /// Get the user IDs for the given room with the given memberships and
     /// stripped state.
-    ///
-    /// If `memberships` is empty, returns all user IDs in the room with the
-    /// given stripped state.
-    fn get_user_ids_inner(
+    fn get_user_ids(
```

Comment:
Can you rename this back to have the `_inner` suffix so the name doesn't conflict with the trait method? It's internal to this module anyways, so doesn't need a "pretty" name.